web/src/composebox_typeahead.ts

```diff
@@ -613,6 +613,7 @@ type PersonSuggestionOpts = {
 export function get_person_suggestions(
     query: string,
     opts: PersonSuggestionOpts,
+    exclude_non_welcome_bots?: boolean,
```

Comment:
We can default this to `false`. Would be more readable.
```typescript
exclude_non_welcome_bots = false;
```

Reply:
Done!